webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
```

Comment:
Since your app is still in development, please add the development mode to your `webpack.config.js`. To do so, you can add the following line:
```js
module.exports = {
  mode: 'development', // HERE
  entry: './src/index.js',
```
[Check this link ](https://cloud.google.com/looker/docs/dev-mode-prod-mode)if you are interested in knowing more about production/dev mode 😉.

Reply:
okay thank you 👍